schedule/yast/yast2_ncurses_gnome.yaml

```diff
@@ -48,3 +52,4 @@ conditional_schedule:
                 - console/yast2_dns_server
                 - console/yast2_nfs_client
                 - console/yast2_snapper_ncurses
+                - x11/yast2_snapper
```

Comment:
All of them can be one change after the L12. 
Although you should not add the` x11/yast2_snapper`. From the description of the test "Test for yast2 UI, ncurses only. Test suite requires HDD settings to be set, we use inheritance for that." And it does not make sense IMO also to run it here as the exact module before that it does quite the same test. I would suggest to put it in the (yast2_gui)[https://openqa.suse.de/tests/4296394]

Reply:
> All of them can be one change after the L12.
I thought I should follow Joaquin's example and add it in "yast_modules". 

> Although you should not add the` x11/yast2_snapper`. From the description of the test "Test for yast2 UI, ncurses only. Test suite requires HDD settings to be set, we use inheritance for that." And it does not make sense IMO also to run it here as the exact module before that it does quite the same test. I would suggest to put it in the (yast2_gui)[https://openqa.suse.de/tests/4296394]

That was what I wanted to do but yast2_gui runs only on x86 and I was told that enabling it for other archs is out of the scope of this ticket.

Reply:
> I thought I should follow Joaquin's example and add it in "yast_modules".

The reason for his approach is that the schedule is diverged. you dont have such case. 

> That was what I wanted to do but yast2_gui runs only on x86 and I was told that enabling it for other archs is out of the scope of this ticket.

If this is out of scope then just do it for x86. but i really do not believe that it belongs here. As per the description of the suite and the logic.

Reply:
> > All of them can be one change after the L12.
> 
> I thought I should follow Joaquin's example and add it in "yast_modules".
> 
> > Although you should not add the` x11/yast2_snapper`. From the description of the test "Test for yast2 UI, ncurses only. Test suite requires HDD settings to be set, we use inheritance for that." And it does not make sense IMO also to run it here as the exact module before that it does quite the same test. I would suggest to put it in the (yast2_gui)[https://openqa.suse.de/tests/4296394]
> 
> That was what I wanted to do but yast2_gui runs only on x86 and I was told that enabling it for other archs is out of the scope of this ticket.

I did the same for ncurses_gnome, I extended the coverage of architecture, but I don't extend the scope of the modules, so you don't need to deal with failures of other yast2_gui tests in the scope of this ticket. Therefore yast2_gui is more suitable.